.travis.yml

```diff
@@ -18,7 +18,8 @@
 sudo: required
 language: python
 install:
-  - pip install --upgrade Cython trappy bart-py devlib psutil wrapt ipython jupyter matplotlib
+  # Use IPython 5.x because 6.0+ only supports Python 3.3
```

Comment:
Yeah :( https://github.com/jupyter/roadmap/blob/master/accepted/migration-to-python-3-only.md